activiti-core/activiti-engine/pom.xml

```diff
@@ -201,6 +216,33 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.1.1</version>
+        <executions>
+          <execution>
+            <id>resource-dependencies</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.activiti</groupId>
+                  <artifactId>activiti-engine</artifactId>
+                  <version>7.2.0</version>
```

Comment:
What do you think about using a variable with a meaningful name(e.g. activiti.previous.version)?